net-im/slack-bin/slack-bin-2.3.3.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit unpacker gnome2-utils
+
+QA_PREBUILT="
+	opt/slack/slack
+	opt/slack/resources/app.asar.unpacked/node_modules/*
+	opt/slack/libnode.so
+	opt/slack/libgcrypt.so.11
+	opt/slack/libffmpeg.so
+	opt/slack/libCallsCore.so
+"
+
+DESCRIPTION="Team collaboration tool"
+HOMEPAGE="http://www.slack.com/"
+
+MY_PN="${PN/-bin/}"
+BASE_URI="https://downloads.slack-edge.com/linux_releases/${MY_PN}-desktop-${PV}-_arch_.deb"
+
+SRC_URI="
+	amd64? ( ${BASE_URI/_arch_/amd64} )
+"
+
+LICENSE="MIT Apache-2.0 BSD ISC LGPL-2 AFL-2.1 public-domain WTFPL-2 Artistic-2 no-source-code"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+RESTRICT="mirror"
+
+RDEPEND="x11-libs/gtk+:2
+	x11-libs/libnotify
+	x11-libs/libXtst
+	x11-libs/pango
+	x11-libs/cairo[xcb]
+	media-libs/alsa-lib
+	media-libs/harfbuzz[graphite]
```

Comment:
harfbuzz is subslotted, so it may be a good idea to state the required subslot here, i.e. matching the SONAME used upstream. The same applies to other packages with subslots.